deploy/kubernetes/kubernetes.go

```diff
@@ -61,7 +53,7 @@ func (k *K8S) Deploy(
 	// be created automatically, then incoming traffic to Service will be forward to Pod.
 	// Then we have no need to create Endpoint manually anymore.
 	selector := map[string]string{
-		"app": "fx-app-" + uuid.New().String(),
+		"app": "fx-app-" + name,
```

Comment:
Maybe we should check if a name is used here

Reply:
Deployment name is guaranteed to be unique by [Line 60](https://github.com/polyrabbit/fx/blob/f92491d7e0a9d22df21f238cc619add9eccd3d67/deploy/kubernetes/kubernetes.go#L60). So I presume the name with a `"fx-app-"` prefix should be unique too.

To be precise, Yes, we need more check here.